docs/content/manual/advanced/addons/5846-vm-import-enhancement.md

```diff
@@ -0,0 +1,453 @@
+---
+title: VM import with EFI mode and secure boot
+---
+
+* Related issues: [#5846](https://github.com/harvester/harvester/issues/5846) [ENHANCEMENT] vm-import-controller enhancements
+
+## Category: 
+* Virtual Machine
+
+## OpenStack Prerequisite Setup
+1. Prepare a baremetal or virtual machine to host the OpenStack service 
+1. Or use the automated Jenkins pipeline to prepare a devstack cluster (Openstack 16.2) (stable/train)
+1. Install OpenStack command line tool on your local machine (introduce in next section)
+
+## VMware vSphere Prerequisite Setup
+1. Get the available access to the existing or prepared vSphere client (v7.1)
+
+## Harvester Prerequisite Setup
+1. Harvester can connect to the OpenStack dashbord and API endpoint
```

Comment:
Typo, dashboard

Reply:
Thanks for the check. Update the typos to dashboard.